one-pager-maker/src/pages/Edit.tsx

```diff
@@ -1,54 +1,100 @@
-import { useState } from 'react'
-import Markdown from 'react-markdown'
 import './Edit.css'
+import { useEffect, useState } from 'react'
+import Markdown from 'react-markdown'
+import { useUpdateDocumentMutation, useFetchDocumentQuery } from "../redux/document/documentsApi.ts";
+import { useNavigate, useParams } from "react-router-dom";
+import { Document, Status } from "../redux/document/documentType.ts";
+import { auth } from '../firebase';
 
 function Edit() {
-  const [markdown, setMarkdown] = useState('');
-  const toMarkdownText = (e:React.ChangeEvent<HTMLTextAreaElement>) => {
-    setMarkdown(e.target.value);
-  };
+  const uid = auth.currentUser?.uid ?? "";
```

Comment:
_This is just an advice, so it is okay to ignore this and do merging._ 

When `currentUser` changes, EditPage might not be re-rendered properly.
So like when user logged out, `auth.currentUser` would become `null`, but EditPage might continue to show user's data.

To resolve this, try to use `useSelector(state => state.user.user.uid)`